test/testsets/Emitters.jl

```diff
@@ -6,6 +6,7 @@ using Test
 using OpticSim
 using OpticSim.Emitters
 using OpticSim.Geometry
+using BenchmarkTools
```

Comment:
I don't think this is needed

Reply:
yeah, that was an accident. I put it in temporarily because it was much faster to verify that the fixes were working than running the test set.

This is a weakness of our current setup. There are many times when you need to run a quick test like this, when you are first debugging the code. If this is done in the test sets then it's a 5 minute cycle as opposed to a few seconds if BenchmarkTools is a dependency. 

I'll remove the BenchmarkTools dependency for now.